bootstrap.d/net-misc.yml

```diff
@@ -3,15 +3,15 @@ packages:
     source:
       subdir: 'ports'
       git: 'https://github.com/curl/curl.git'
-      tag: 'curl-7_76_0'
-      version: '7.76.0'
+      tag: 'curl-7_77_0'
+      version: '7.77.0'
       tools_required:
         - host-autoconf-v2.69
         - host-automake-v1.15
         - host-libtool
         - host-autoconf-archive
       regenerate:
-        - args: ['./buildconf']
+        - args: ['autoreconf', '-fiv']
```

Comment:
Why is this done? Did the upstream package remove that script?

Reply:
On running `./buildconf`, a warning message is printed stating that one should use `autoreconf -fiv` instead. I assume `./buildconf` is deprecated upstream.